examples/python/detect_and_track_objects/detect_and_track_objects.py

```diff
@@ -7,9 +7,10 @@
 import json
 import logging
 import os
+from collections.abc import Sequence
```

Comment:
why are some of these moved from `typing` to `collection.abc`? Is there a lint to catch the wrong usage?